spaceship/lib/spaceship/tunes/application.rb

```diff
@@ -136,6 +137,16 @@ def url
         "https://itunesconnect.apple.com/WebObjects/iTunesConnect.woa/ra/ng/app/#{self.apple_id}"
       end
 
+      def analytics
+        if self.latest_version.app_status != AppStatus::READY_FOR_SALE
```

Comment:
I think this if statement might be too strong? Event if the latest version of the app isn't ready for sale, there could be analytics from a previous version that is currently live. Right? 🤔 

I _think_ we want to make sure `live_version` isn't nil?